src/shopify/utils/oauth_handler.bal

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) 2020 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we rename this class for `oauth2_handler.bal`, since Shopify uses OAuth2 and not OAuth1.0a?